terraform/modules/service/variables.tf

```diff
@@ -72,3 +72,15 @@ variable "app_cpu" {
 variable "app_memory" {
   type = number
 }
+
+variable "tcp_healthcheck" {
+  # TODO: Remove this when all services use HTTP healthcheck
+  type    = bool
+  default = true
+}
+
+variable "healthcheck_path" {
+  # Note: this is only used when tcp_healthcheck is set to false
```

Comment:
do you need both variables? ie TCP healthcheck if no path is specified, HTTP otherwise

Reply:
I think if we want a default healthcheck path (which is where i'd like to end up), we need 2 variables.